src/main/java/frc/robot/subsystems/Ramp.java

```diff
@@ -38,35 +38,23 @@ public void periodic() {
             SmartShuffleboard.put("Ramp", "Desired pos", rampPos);
             SmartShuffleboard.put("Ramp", "Reverse Switch Tripped", getReversedSwitchState());
             SmartShuffleboard.put("Ramp", "Forward Switch Tripped", getForwardSwitchState());
-//            double pidP = SmartShuffleboard.getDouble("Ramp", "PID P", neoPidMotor.getPidController().getP());
-//            double pidI = SmartShuffleboard.getDouble("Ramp", "PID I", neoPidMotor.getPidController().getI());
-//            double pidD = SmartShuffleboard.getDouble("Ramp", "PID D", neoPidMotor.getPidController().getD());
-//            double pidFF = SmartShuffleboard.getDouble("Ramp", "PID FF", neoPidMotor.getPidController().getFF());
-//            if (pidP != neoPidMotor.getPidController().getP()) neoPidMotor.getPidController().setP(pidP);
-//            if (pidI != neoPidMotor.getPidController().getI()) neoPidMotor.getPidController().setI(pidI);
-//            if (pidD != neoPidMotor.getPidController().getD()) neoPidMotor.getPidController().setD(pidD);
-//            if (pidFF != neoPidMotor.getPidController().getFF()) neoPidMotor.getPidController().setFF(pidFF);
-            SmartShuffleboard.put("Ramp", "Forward Switch Tripped", getForwardSwitchState());
```

Comment:
why deleting the debug shuffleboard values?

Reply:
delete selection was to big when removing pid stuff

Reply:
fixed